src/main/resources/assets/botania/lang/en_us.json

```diff
@@ -3334,47 +3334,84 @@
   "botania.entry.preventingDecay": "Numerical Mana",
   "botania.tagline.preventingDecay": "Seeing the numbers in mana",
   "botania.page.preventingDecay0": "no",
-
-  "botania.subtitle.airRod": "Rod of the Skies whooshes",
+  
   "botania.subtitle.agricarnation": "Agricarnation glimmers",
-  "botania.subtitle.altarCraft": "Petal Apothecary finishes",
+  "botania.subtitle.airRod": "Rod of the Skies whooshes",
+  "botania.subtitle.altarCraft": "Petal Apothecary twinkles",
   "botania.subtitle.arcane_rose_disenchant": "Rosa Arcana disenchants",
+  "botania.subtitle.astrolabeConfigure": "Worldshaper's Astrolabe clinks",
   "botania.subtitle.babylonAttack": "Key of King's Law unleashed",
   "botania.subtitle.babylonSpawn": "Key of King's Law activated",
-  "botania.subtitle.bellows": "Manatide Bellows hisses",
+  "botania.subtitle.bellows": "Manatide Bellows hiss",
```

Comment:
should it still be hisses to match all the other ones?

Reply:
oops, missed that one

Reply:
I'll change it back for consistency with the rest of the lexica, but I'm not sure if it's actually correct - I interpreted "bellows" as a plural noun, but other parts of the lexica say things like "The Manatide Bellows is", which sounds wrong to my ears (i would use "bellows are")